Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/Lib/Helpers.cs

```diff
@@ -144,7 +145,8 @@ internal static void CreateSubDirectory(string path)
         using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/33bbc16e411f6fa4f5f8086d85e2a83889335206/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/Lib/Helpers.cs#L105
You don't have to initialize this variable, since you are just setting values for it. You can just declare it with `string current`.

Reply:
Removed it all together. Was left there when I was testing.